hyperspy/io.py

```diff
@@ -478,6 +478,9 @@ def save(filename, signal, overwrite=None, **kwds):
         ensure_directory(filename)
         if overwrite is None:
             overwrite = overwrite_method(filename)
+        if overwrite is False:
```

Comment:
I am a bit confused, are you suggesting replacing lines 481-483 with the above? 

When I test, this PR works correctly for all four combinations: 
1) overwrite = True, file exists --> file is overwritten
2) overwrite = True, file doesn't exist --> new file is written
3) overwrite = False, file exists --> do nothing
4) overwrite = False, file doesn't exist --> new file is written

I am not able to interpret the Travis CI and AppVeyor reports, but if the problem is not doing anything at 3), then raising an error is no better. I was using `save(overwrite=False)` because I wanted no interruptions when looping a process over many files creating multiple output files for each, not knowing whether I had made one of output files before. 

Better would be (starting at line 481): 
```
if overwrite is False:
    if os.path.isfile(filename):
        pass
    else: 
        overwrite = True
```